tests/unit/drm_plugin_test.sh

```diff
@@ -119,6 +121,12 @@ function test_drm_parser_options()
   parse_drm_options --gui-off
   assertEquals "($LINENO)" 1 "${options_values['GUI_OFF']}"
 
+  parse_drm_options --gui-on-after-reboot
+  assertEquals "($LINENO)" 1 "${options_values['GUI_ON_AFTER_REBOOT']}"
+
+  parse_drm_options --gui-off-after-reboot
+  assertEquals "($LINENO)" 1 "${options_values['GUI_OFF_AFTER_REBOOT']}"
```

Comment:
Use {} around `LINENO` Use {} around `LINENO` 
https://kworkflow.org/content/codingstyle.html#string-concatenation

(I know that throughout this file it there is no {}, but this is the correct way).